src/main.ts

```diff
@@ -2,13 +2,22 @@ import * as git from './git';
 import * as installer from './installer';
 import * as core from '@actions/core';
 import * as exec from '@actions/exec';
+import {dirname} from 'path';
 
 async function run(): Promise<void> {
   try {
     const version = core.getInput('version') || 'latest';
-    const args = core.getInput('args', {required: true});
+    const isInstallOnly = core.getInput('install-only') === 'true';
```

Comment:
```suggestion
    const isInstallOnly = /true/i.test(core.getInput('install-only'));
```

Reply:
regex `/true/i` is matching any text containing case-insensitive "true" in it.

```
> /true/i.test("text-containing-true-matches")
true
```

May be we will have to use `/^true$/i`

```
> /^true$/i.test("text-containing-true-matches")
false
```

Reply:
@arunvelsriram LGTM